t/plugin/rocketmq-logger.t

```diff
@@ -0,0 +1,1098 @@
+#
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+use t::APISIX 'no_plan';
+
+repeat_each(1);
+no_long_string();
+no_root_location();
```

Comment:
LGTM. We can improve the test in a separate PR:

The newly added test file should use a preprocessor to reduce duplicate `--- request` & `--- no_error_log` section. You can take a look at:

https://github.com/apache/apisix/blob/11433d740c5e19b757c61f59842d6de64e27fba8/t/plugin/gzip.t#L33=L42

And please split it into rocketmq-logger.t & rocketmq-logger2.t. Each test file should be smaller than 800L.